proxy/http2/HPACK.cc

```diff
@@ -263,12 +263,80 @@ Http2DynamicTable::add_header_field(const MIMEField *field)
     }
 
     MIMEField *new_field = _mhdr->field_create(name, name_len);
+    new_field->name_set(_mhdr->m_heap, _mhdr->m_mime, name, name_len);
     new_field->value_set(_mhdr->m_heap, _mhdr->m_mime, value, value_len);
+    mime_hdr_field_attach(_mhdr->m_mime, new_field, 1, NULL);
```

Comment:
Why don't you use `MIMEHdr::field_attach`? It's inconsistent to use the function here.